cloudscope/xunit.py

```diff
@@ -18,62 +21,70 @@ def __init__(self, project, user, password, jobUrl, lastBuildNumber = 0):
         self.lastBuildNumber = lastBuildNumber
         self.es = Elasticsearch()
 
-        #TODO put this json in seprate file
-        xunitMapping = { "mappings": {
-                            "testjob": {
-                                "properties": {
-                                    "duration": {
-                                    "type": "long"
-                                    },
-                                    "id": {
-                                    "type": "long"
-                                    },
-                                    "name": {
-                                    "type": "string"
-                                    },
-                                    "result": {
-                                    "type": "string"
-                                    },
-                                    "time": {
-                                    "type": "date"
-                                    }
-                                }
-                            },
-                            "testsuite": {
-                                "_parent": {
-                                    "type": "testjob"
-                                }
-                            },
-                            "testcase": {
-                                "_parent": {
-                                    "type": "testsuite"
-                                }
-                            }
-                        }
+        xunitMapping = {"mappings": {
```

Comment:
Can we change all the variable and function names to not be camel case and be separated by underscores.